.github/workflows/cd.yaml

```diff
@@ -26,21 +26,20 @@ jobs:
         run: ./gradlew :launcher:shadowJar
         working-directory: ${{ runner.temp }}/RegistrationService
 
+      - name: 'Copy Registration Service CLI'
+        run: |
+          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.registrationservice:registration-service-cli:$REGISTRATION_SERVICE_VERSION:jar:all -DoutputDirectory=.
```

Comment:
what's the reason for using `mvn` here instead of a file copy? Since we're building the RegSrv in the step above, that should be possible , no? Or did I overlook something?

Reply:
This is to be future-proof for when the RS will be consumed as a maven artifact.

Reply:
there is an alternative that uses Gradle: https://stackoverflow.com/a/26698161

Reply:
not for now, but I would say if a runnable artifact is needed for RegistrationService, then it should be available for download (e.g. using `wget`).
Alternatively, MVD could define a launcher for the RegistrationService, which is used to compile the binary.